carma_base/package.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!--
+  Copyright (C) 2018 LEIDOS.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<package>
+    <name>carma_base</name>
+    <version>2.8.0</version>
```

Comment:
You know we've released 2.8.1, right?  Do you have a good handle on which files will need to be updated with each new version ID?  We need to write up a release process doc, and those details should be captured there.  Please send me any notes on the topic.